libs/payments/paypal/src/lib/paypal.manager.ts

```diff
@@ -3,10 +3,21 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { Injectable } from '@nestjs/common';
-import { PayPalClient } from './paypal.client';
 import { AccountDatabase } from '@fxa/shared/db/mysql/account';
+import { PayPalClient } from './paypal.client';
 
 @Injectable()
 export class PayPalManager {
   constructor(private db: AccountDatabase, private client: PayPalClient) {}
+
+  /**
+   * Get a token authorizing transaction to move to the next stage.
+   *
+   * If the call to PayPal fails, a PayPalClientError will be thrown.
+   *
+   */
+  async getCheckoutToken(options: { currencyCode: string }) {
```

Comment:
Since this is a single param I'd just have it be a single argument rather than an options object